gradio/components/image_editor.py

```diff
@@ -116,8 +116,8 @@ def __init__(
         data_mode: Literal["image", "pathline"] = "image",
         crop_size: tuple[int | float, int | float] | str | None = None,
         transforms: Iterable[Literal["crop", "rotate"]] = ("crop", "rotate"),
-        eraser: Eraser | None = Eraser(),
-        brush: Brush | None = Brush(),
```

Comment:
This is not recommended as the objects will only be instantiated once per Python session, even if you have multiple classes

Reply:
Maybe we can use `False` for that case.